packages/rollup-plugin-import-meta-assets/test/integration.test.js

```diff
@@ -87,6 +87,34 @@ describe('rollup-plugin-import-meta-assets', () => {
     ]);
   });
 
+  it('simple bundle with ignored assets', async () => {
+    const config = {
+      input: { 'transform-bundle-ignored': require.resolve('./fixtures/transform-entrypoint.js') },
+      plugins: [
+        importMetaAssets({
+          transform: async (assetBuffer, assetPath) => {
+            // Only minify SVG files
+            return assetPath.endsWith('.svg')
+              ? // Fake minification with an XML comment
+                assetBuffer.toString() + '<!-- minified -->\n'
+              : null;
```

Comment:
what would be the benefit of supporting `null`? is that significantly faster than returning the buffer? 🤔

e.g. how does it compare with
```js
return assetPath.endsWith('.svg') ? assetBuffer.toString() + '<!-- minified -->\n' : assetBuffer;
```

Reply:
ah I now understand your use case - makes sense 👍

Reply:
Sorry hadn't had the chance to reply yet. This would mostly be useful for denoting you don't want the buffer to be processed at all. E.g. if you are gradually migrating to this plugin, or if you still have some images left that are pre-processed differently. For example, compressing with AVIF can take quite a bit of time, depending on your compression settings. Therefore it doesn't always make sense to compress every single image during build time and instead do that as a pre-processing step.